onnx/onnx.in.proto

```diff
@@ -543,14 +543,14 @@ message TypeProto {
   }; 
 
   message Record { 
-      // Domain with empty string means the model's domain. 
-      optional string domain = 1; 
-      // The name is required for this version of the IR.
-      optional string name = 2; 
+    // Domain with empty string means the model's domain. 
```

Comment:
"empty string" or "missing domain"? (Same comment applies to other place where domain is used as well.)